test/e2e/vpcs.e2e.ts

```diff
@@ -41,6 +41,35 @@ test('can nav to VpcPage from /', async ({ page }) => {
   await expect(page.getByRole('cell', { name: 'allow-icmp' })).toBeVisible()
 })
 
+test('can edit VPC', async ({ page }) => {
+  // update the VPC name, starting from the VPCs list page
+  await page.goto('/projects/mock-project/vpcs')
+  await expectRowVisible(page.getByRole('table'), { name: 'mock-vpc' })
+  await clickRowAction(page, 'mock-vpc', 'Edit')
+  await expect(page).toHaveURL('/projects/mock-project/vpcs/mock-vpc/edit')
+  await page.getByRole('textbox', { name: 'Name' }).first().fill('mock-vpc-2')
+  await page.getByRole('button', { name: 'Update VPC' }).click()
+  await expect(page).toHaveURL('/projects/mock-project/vpcs/mock-vpc-2/firewall-rules')
+  await expect(page.getByRole('heading', { name: 'mock-vpc-2' })).toBeVisible()
+
+  // now update the VPC description, starting from the VPC view page
+  await page.getByRole('button', { name: 'VPC actions' }).click()
+  await page.getByRole('menuitem', { name: 'Edit' }).click()
+  await expect(page).toHaveURL('/projects/mock-project/vpcs/mock-vpc-2/edit')
+  await page.getByRole('textbox', { name: 'Description' }).fill('updated description')
+  await page.getByRole('button', { name: 'Update VPC' }).click()
+  await expect(page).toHaveURL('/projects/mock-project/vpcs/mock-vpc-2/firewall-rules')
```

Comment:
You can assert the description here since on this one we do have the properties table.